docs/src/main/tut/client.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: HTTP Client
+---
+
+## Your first client
+
+How do we know the server is running?  Let's create a client with
+http4s to try our service.
+
+### Creating the client
+
+A good default choice is the `PooledHttp1Client`.  As the name
+implies, the `PooledHttp1Client` maintains a connection pool and
+speaks HTTP 1.x.
+
+```tut:book
+import org.http4s.client.blaze._
+
+val httpClient = PooledHttp1Client()
+```
+
+### Describing a call
+
+To execute a GET request, we can call `getAs` with the type we expect
+and the URI we want:
+
+```tut:book
+val helloJames = httpClient.expect[String]("http://localhost:8080/hello/James")
+```
+
+Note that we don't have any output yet.  We have a `Task[String]`, to
+represent the asynchronous nature of a client request.
+
+Furthermore, we haven't even executed the request yet.  A significant
+difference between a `Task` and a `scala.concurrent.Future` is that a
+`Future` starts running immediately on its implicit execution context,
```

Comment:
Pedantry: a `Future` doesn't have an execution context.  It is implicitly passed one in certain constructions, and again on certain operations.  This distinction has caused battles on Twitter. :(